Dockerfile.cross

```diff
@@ -0,0 +1,21 @@
+# This image is meant to enable cross-architecture builds.
+# It assumes the foundry binaries have already been compiled for `$TARGETPLATFORM` and are
+# locatable in `./dist/bin/$TARGETARCH`
+FROM ubuntu:22.04
```

Comment:
any reason to use ubuntu instead alpine?

Reply:
Because we use `cross` to build the binary in the host, and then use `docker copy` into the container, so it need to keep the same arch. Otherwise these binaries will not work due to the absence of library